packages/ckeditor5-paste-from-office/tests/_data/list/many-one-item/normalized.word2016.html

```diff
@@ -1,113 +1,39 @@
 <ol>
-	<li class=MsoListParagraphCxSpFirst style='mso-list:l2 level1 lfo1'>
-		<![if !supportLists]>
```

Comment:
Is this correct that this part is missing?

Reply:
I guess yeah, because content usually is in span element and `<![if !supportLists]>` doesn't look like content in my opinion.

Reply:
I also think that it could be removed - looks like it is some kind of leftover after MSIE.